### DISCUSSION TITLE
An unexpected error occurred. Please try again.

### DISCUSSION THREAD

**Discussion by ImLunaHey** [planetscale/discussion#453] [Bug Reports]
Leaving a tab in the background and going back to it causes this error to show for the first few attempts. Refreshing resolves this.

![image](https://user-images.githubusercontent.com/105922175/235070586-47025e8e-b052-4ce6-aedd-74a019bc541d.png)

**Comment by mscoutermarsh** [comment]
Thanks for sharing!

This happens because the DB connection used by our web console has expired & the UI isn't renewing it fast enough. We're working on fixing it (we use [database.js](https://github.com/planetscale/database-js) to establish a connection from your browser to your database).

**Reply by ImLunaHey** [reply]
So the weird thing is some times it'll fail, fail, work, fail, work, work, work...

Same issue? 🤔

**Reply by mscoutermarsh** [reply]
Are you leaving the page for a while and then coming back?
